### ISSUE TITLE
Bad performance of swizzling `nxEV_layoutSubviews`

### ISSUE THREAD

**Issue by ludwigschubert** [nxtbgthng/UITableView-NXEmptyView#13]
I'm not too sure why this is a problem, but Instruments showed me that 90+% of computation in my table view during simply scrolling was spent in the swizzled `nxEV_layoutSubviews` method. 

Sorry I couldn't investigate further, but thought it was worth letting you know. Thanks in concept for the extension, though! :D

**Comment by markgaensicke** [comment]
Yes, I think this is because UIKit very frequently calls layoutSubviews during scrolling. The swizzled layoutSubviews method then calls nxEV_updateEmptyView which leads to the performance problems you've mentioned.
